src/screens/LoginScreen.js

```diff
@@ -0,0 +1,127 @@
+import { useNavigation } from '@react-navigation/native';
+import React, { useContext, useState } from 'react';
+import {
+  SafeAreaView,
+  Text,
+  StyleSheet,
+  TextInput,
+  TouchableOpacity,
+  ActivityIndicator,
+  View,
+} from 'react-native';
+import Apis from '../apis';
+import server from '../configs/server';
+import AuthContext from '../contexts/AuthContext';
+
+const styles = StyleSheet.create({
+  container: {
+    paddingHorizontal: 20,
+    flex: 1,
+    justifyContent: 'center',
+  },
+  header: {
+    fontSize: 25,
+    fontWeight: '700',
+    textAlign: 'center',
+  },
+  input: {
+    borderWidth: StyleSheet.hairlineWidth,
+    borderColor: '#777',
+    marginTop: 15,
+    padding: 5,
+    fontSize: 14,
+    borderRadius: 5,
+  },
+  errorText: {
+    color: 'red',
+    marginTop: 15,
+  },
+  button: {
+    backgroundColor: '#327de6',
+    marginTop: 30,
+    padding: 15,
+    borderRadius: 10,
+    width: 150,
+    alignSelf: 'center',
+  },
+  buttonText: {
+    fontSize: 14,
+    textAlign: 'center',
+    fontWeight: '500',
+    color: 'white',
+  },
+});
+
+function LoginScreen() {
+  const { setAuthentication } = useContext(AuthContext);
+  const navigation = useNavigation();
+  const [inputs, setInputs] = useState({
```

Comment:
use const [user, setUser]

Reply:
Do you mean we should change the name of the variable (like [inputs, setInputs] to [user, setUser])?

Reply:
yes, should group the fields in to domain objects.